test/helpers/increaseTime.js

```diff
@@ -43,5 +43,7 @@ export const duration = {
   minutes: function(val) { return val * this.seconds(60) },
   hours:   function(val) { return val * this.minutes(60) },
   days:    function(val) { return val * this.hours(24) },
-  weeks:   function(val) { return val * this.days(7) }
+  weeks:   function(val) { return val * this.days(7) },
+  months:  function(val) { return val * this.days(30)},
```

Comment:
I'm thinking we should only provide time units matching [those of Solidity](https://solidity.readthedocs.io/en/develop/units-and-global-variables.html#time-units). `months` is not included and I see you haven't used it anywhere either. What do you think about removing it?